operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [ENHANCEMENT] Upgrade nginx image tag to `nginxinc/nginx-unprivileged:1.22-alpine`. #2742
 * [ENHANCEMENT] Upgrade minio subchart to `4.0.12`. #2759
 * [BUGFIX] Do not use undocumented `mulf` function in templates. #2752
+* [BUGFIX] Let grpc connection rebalance after scale out. #2862
```

Comment:
I think this can use a bit more context 

```suggestion
* [BUGFIX] Periodically rebalance gRPC connection between GEM gateway and distributors after scale out of the distributors. #2862
```